odoo/tools/misc.py

```diff
@@ -1380,6 +1380,26 @@ def _format_time_ago(env, time_delta, lang_code=False, add_direction=True):
     return babel.dates.format_timedelta(-time_delta, add_direction=add_direction, locale=locale)
 
 
+def format_number(number, decimal=1):
+    for unit in ['', 'k', 'M', 'G']:
+        if abs(number) < 1000.0:
+            return "%g%s" % (round(number, decimal), unit)
+        number /= 1000.0
+    return "%g%s" % (round(number, decimal), 'T')
+
+
+def format_human_readable_amount(amount, currency=None):
```

Comment:
format_decimalized_amount ? that way we are coherent with format_decimalized_number

Obviously if you find another name for format_decimalized_number, this one could be updated accordingly.

cc @mart-e because I love pinging people several times